tests/jbuild

```diff
@@ -3,4 +3,4 @@
 (alias
   ((name runtest)
    (deps ((files_recursively_in .) ../src/client/opamMain.exe ../src/tools/opam_check.exe))
-   (action (run make all))))
+   (action (run gmake all))))
```

Comment:
Is it a change you wanted to push? It seems a local-BSD modification, not [portable](https://travis-ci.org/ocaml/opam/jobs/435208178#L3219).